src/org/exist/start/start.config

```diff
@@ -104,10 +104,4 @@ tools/jetty/lib/*                                  mode == standalone
 tools/jetty/lib/*                                  mode == other
 tools/ircbot/lib/*                                 mode == jetty
 tools/aspectj/lib/*                                always
-tools/yajsw/wrapperApp.jar                         mode == other
-tools/yajsw/wrapper.jar                            mode == other
-tools/yajsw/lib/extended/velocity/velocity-%latest%.jar         mode == other
-tools/yajsw/lib/core/commons/commons-configuration2-%latest%.jar mode == other
-tools/yajsw/lib/core/commons/commons-lang3-%latest%.jar          mode == other
-tools/yajsw/lib/core/netty/netty-all-4.0.35.Final.jar           mode == other
-tools/yajsw/lib/core/jna/jna-platform-%latest%.jar              mode == other
+tools/yajsw/lib/core/commons/commons-configuration2-%latest%.jar mode == other
```

Comment:
This is not correct. I will send a PR. Unfortunately this means 3.4.0 is somewhat broken for developers, i.e. it won't work in Eclipse or NetBeans or outside of YAJSW (i.e. when `mode == other`). It also has an affect on the Maven artifacts, as they were not updated to reflect this change either.

I think we will need a 3.4.1... :-/